packages/api/src/mutant/Mutator.ts

```diff
@@ -1,6 +1,7 @@
 import { File } from '../../core';
 import Mutant from './Mutant';
+import { ParserPlugin } from '@babel/parser';
 
 export default interface Mutator {
-  mutate(inputFiles: ReadonlyArray<File>): ReadonlyArray<Mutant>;
+  mutate(inputFiles: ReadonlyArray<File>, options?: ParserPlugin[]): ReadonlyArray<Mutant>;
```

Comment:
Instead of changing the api of a mutator, please let the `StrykerOptions` be injected in the `JavaScriptMutator` here: https://github.com/stryker-mutator/stryker/blob/master/packages/javascript-mutator/src/JavaScriptMutator.ts#L13-L16. If you want, I can add this for you, so you can just use it there.